lib/Unicode/Char/Numeric.hs

```diff
@@ -41,3 +45,15 @@ isNumber c = case generalCategory c of
     LetterNumber  -> True
     OtherNumber   -> True
     _             -> False
+
+
+-- | Numeric value of a character, if relevant.
+{-# INLINE numericValue #-}
+numericValue :: Char -> Maybe Rational
+numericValue = V.numericValue
+
+-- | Integer value of a character, if relevant.
+intValue :: Char -> Maybe Int
+intValue c = V.numericValue c >>= \r -> if denominator r == 1
```

Comment:
Would be better if the `if` statement is on the next line. Even better if a `do` block is used.

Reply:
We can use the longer name `integerValue` instead, it is no longer than `numericValue` anyway.

Reply:
OK. I think I chose it for the resulting type.